v2/emailpassword/common-customizations/sessions/token-transfer-method.mdx

```diff
@@ -0,0 +1,314 @@
+---
+id: token-transfer-method
+title: Token transfer methods
+hide_title: true
+show_ui_switcher: true
+---
+
+<!-- COPY DOCS -->
+<!-- ./session/common-customizations/sessions/token-transfer-method.mdx -->
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import TabItem from '@theme/TabItem';
+import NpmOrScriptTabs from "/src/components/tabs/NpmOrScriptTabs"
+import AngularUIImplementation from "/src/components/reusableSnippets/angularUIImplementation"
+import {Answer}from "/src/components/question"
+import {PreBuiltOrCustomUISwitcher, PreBuiltUIContent, CustomUIContent} from "/src/components/preBuiltOrCustomUISwitcher"
+import FrontendPreBuiltUITabs from "/src/components/tabs/FrontendPreBuiltUITabs"
+import FrontendCustomUITabs from "/src/components/tabs/FrontendCustomUITabs"
+import FrontendMobileSubTabs from "/src/components/tabs/FrontendMobileSubTabs"
+
+<!-- TODO: re-think title -->
+# Token transfer methods
+
+:::caution
+We recommend cookie-based sessions in browsers because header-based sessions require saving the access and refresh tokens in storage vulnerable to XSS attacks.
+:::
+
+When creating/authorising sessions, the SDK has to choose to send the tokens to the frontend by cookies or custom headers. This choice is ultimately controlled by the backend, but it follows a preference set in the frontend configuration.
+
```

Comment:
I think a better intro is needed:
- SuperTokens supports cookie based and header based tokens. Header based implies that the `authorization bearer` header will be used when making API calls, and that the session tokens can be read from the frontend.
- By default, for browsers, we use cookie based sessions (prevent against token theft via XSS) and for other clients like mobile apps, we use header based.

We should also talk about the motivation for this:
- Mobile devices have buggy cookie handling
- Authorization Bearer tokens make it easier to work with API Gateways and third party services.
- what else??